src/node_crypto.cc

```diff
@@ -3136,7 +3115,7 @@ static ParseKeyResult ParsePrivateKey(EVPKeyPointer* pkey,
                                       const PrivateKeyEncodingConfig& config,
                                       const char* key,
                                       size_t key_len) {
-  PasswordCallbackInfo pc_info(config.passphrase_.get());
+  char* passphrase = const_cast<char*>(config.passphrase_.get());
```

Comment:
maybe a comment saying why const has to be cast away?

Reply:
And maybe `char* const passphrase = ...` to clarify it isn't reassigned? The use sites are quite far away from here.